site/docs/guidelines/inclusive_design.mdx

```diff
@@ -0,0 +1,70 @@
+
+In Helsinki, we aim to apply inclusive design in all parts of the service development. Not only this means designing, developing, testing, and releasing the service, but also everything related to it such as meetings, workshops, and daily interactions with other people. 
+
+As stated in the Helsinki equality plan, "_The most important goal of all gender equality work is to make Helsinki develop into a city where people representing all genders, gender identities, and gender expressions can be seen and heard and are allowed to participate and act as equal citizens without direct or indirect discrimination_". 
+
+## Setting up your project for success
+When starting a new project, there are two (2) main elements that it comes to ensure inclusivity. 
+
+### 1. Multidisciplinary and diverse team
+The structure of your project team shapes the decision it makes. Always aim to have a multidisciplinary team and make sure your team members can work together. It is ideal if all team members can be part of the project right from the research and design phase to the testing and release.
+
+It is not always possible to ensure that your team is diverse enough but it is important to acknowledge the diversity of your team. And especially in small non diverse teams, it is important to make sure the designs and implementations are tested with a more diverse group of test users.
+
+### 2. Involving users
+<p>To create services with a good user experience and accessibility, it is essential to involve real users. Plan early how you will involve real users during your project. Make sure users involved in research and user testing are chosen from people from varying backgrounds. You can find more information about research and user testing in projects in the <Link href="https://pelikirja.hel.fi/" external>Helsinki Playbook (in Finnish)</Link>.</p>
+
+## Best practises
+While designing inclusive and accessible services may seem a daunting task, it gets easier when you take it into account from the beginning of the project. Here is a list of best practises from the city of Helsinki projects.
+
+### Imagery
+Most services in Helsinki use some kind of imagery either for conveying information or adding visual interest. Using imagery always requires some extra thought to ensure all users can feel welcomed and included in your service.
+
+If you are using photographs, make sure you intentionally diversify your images. This means ensuring that a wide array of different users are represented in your imagery.
+
+<p>Using abstract images is another way of improving the inclusiveness of your images. Instead of using realistic and detailed photographs, your project can use more conceptual illustrations. <Link href="https://brand.hel.fi/kuvitusohjeisto/" external>The Helsinki Visual identity guideline</Link> has a wide array of abstract image materials to be used in your project.</p>
```

Comment:
```<p>```- tags can be omitted

Reply:
On the current documentation site you cannot. Using the <Link> component requires <p> tags - otherwise anything below it won't render.